modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/SkinMemoryLeakTest.java

```diff
@@ -211,6 +221,7 @@ public SkinMemoryLeakTest(Class<Control> controlClass) {
      * @param focused if true, requests focus on the added control
      */
     protected void showControl(Control control, boolean focused) {
+        // use StageLoader maybe?
         if (root == null) {
```

Comment:
we could, but don't here for consistency - the pattern is used across multiple tests to guarantee full control about the layout (which conceededly is not needed here :)